steam_wrapper/steam_wrapper.py

```diff
@@ -91,13 +91,16 @@ def copytree(source, target, ignore=None):
             os.utime(full_target)
 
 def check_bad_filesystem_entries(entries):
-    bad_names = ["home", "host", os.path.expandvars("/var/home/$USER"),
-         "/home/$USER"]
+    bad_names = ["home",
+                 "host",
+                 os.path.expandvars("/var/home/$USER"),
+                 os.path.expandvars("/home/$USER")]
+    bad_topdirs = ["xdg-config", "xdg-data", "xdg-cache"]
     found = False
     for entry in entries:
-        items = entry.split(";")
```

Comment:
I don't know why did you re-split `entry` here, since is should be already spitted by `;` in `read_flatpak_info()`, so I used `entry` var as is, but added an assertion just in case.

Reply:
Probablu leftovers